cmd/cmd.go

```diff
@@ -59,6 +60,24 @@ func GetMeshConfig(kube kubernetes.Interface, namespace, name string) (*proxycon
 		return nil, multierror.Prefix(err, "failed to convert to proto.")
 	}
 
+	if mesh.MixerAddress != "" && mesh.StatsdUdpAddress == "" {
+		colon := strings.Index(mesh.MixerAddress, ":")
+		mixerAddr := mesh.MixerAddress[:colon]
+		if err := model.ValidateIPv4Address(mixerAddr); err == nil {
+			mesh.StatsdUdpAddress = mesh.MixerAddress
+		} else {
+			if mixerSvc, err := kube.CoreV1().Services(namespace).Get(mixerAddr, v1.GetOptions{}); err == nil {
```

Comment:
Can you do DNS resolution yourself here? Please don't fallback onto mixer address though, we should just change the value of statsd address to IP:PORT until Envoy gets DNS support for statsd setting. You can use go's `LookupHost` to parse out and replace host.